### DISCUSSION TITLE
quasar dev -m -electorn fail

### DISCUSSION THREAD

**Discussion by 332lfgb6** [quasarframework/quasar#9285] [CLI - Electron mode] [answered]
![image](https://user-images.githubusercontent.com/51245335/118363068-3823ff00-b5c5-11eb-8129-8db04f9f7169.png)
![image](https://user-images.githubusercontent.com/51245335/118363077-42de9400-b5c5-11eb-87f3-4f3befbe10c7.png)
> What should I do?

**Comment by hawkeye64** [comment, marked as answer]
Looks like you hit the Windows 10 error: https://github.com/electron/electron/issues/15554#issuecomment-840351054

**Reply by 332lfgb6** [reply]
I solved this problem by rebooting the computer
